modules/Bio/EnsEMBL/Compara/RunnableDB/GeneTrees/DumpWGAExpected.pm

```diff
@@ -0,0 +1,87 @@
+=head1 LICENSE
+
+See the NOTICE file distributed with this work for additional information
+regarding copyright ownership.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 CONTACT
+
+  Please email comments or questions to the public Ensembl
+  developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+  Questions may also be sent to the Ensembl help desk at
+  <http://www.ensembl.org/Help/Contact>.
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::RunnableDB::GeneTrees::DumpWGAExpected
+
+=head1 SYNOPSIS
+
+A runnable to dump wga_expected tag and delete it from a database.
+
+=cut
+
+package Bio::EnsEMBL::Compara::RunnableDB::GeneTrees::DumpWGAExpected;
+
+use strict;
+use warnings;
+
+use base ('Bio::EnsEMBL::Compara::RunnableDB::BaseRunnable');
+
+
+sub run {
+    my ($self) = shift @_;
+   
+    $self->dump_wga_expected();
+
+    $self->delete_wga_expected();
```

Comment:
I would personally move the code of these two methods here, since it is small enough.

Reply:
I see. Let's see what @carlacummins thinks.